go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aquasecurity/trivy
 go 1.13
 
 require (
-	github.com/aquasecurity/fanal v0.0.0-20200505074551-9239a362deca
+	github.com/aquasecurity/fanal v0.0.0-20200526073652-33a588b6fcba
```

Comment:
You will update this to master before you merge this PR right?

Reply:
It's fine I had to do this update since I needed to build this locally https://github.com/aquasecurity/trivy/pull/503/commits/2f89c0b60a08a183f9cb510f5c4feeee3c276d6f

Reply:
Thanks!